packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -302,7 +301,10 @@ class EditorPage {
 	async waitForKeyboardToBeHidden() {
 		const { addButtonLocation } = this.initialValues;
 		const addButton = await this.getAddBlockButton();
-		const location = await addButton.getLocation();
```

Comment:
Occasionally, this query would fail with Appium 2 with the following error. Guarding against an empty query return avoids this error and results in the recursive loop of `waitForKeyboardToBeHidden` to occur. 

```
TypeError: Cannot read properties of undefined (reading 'getLocation')
```